autoscheduler/scraper/models/instructor.py

```diff
@@ -2,7 +2,7 @@
 
 class Instructor(models.Model):
     """ Model that represents a professor, who may teach many courses """
-    id = models.IntegerField(primary_key=True)
+    id = models.CharField(max_length=100,primary_key=True)
     email_address = models.CharField(max_length=48, null=True)
     name = models.CharField(max_length=64)

```

Comment:
Since we're  going to use the name for the primary key for now, the id's max length should match the name's, and name should be removed.